component/rating/rating.css

```diff
@@ -0,0 +1,21 @@
+.rating-container{
+    display: flex;
+    gap:0.1rem;
```

Comment:
Use px for very small length.

Reply:
I just want to be consistent throughout the code !!